supabase/migrations/20230930062908_drop_constraint.sql

```diff
@@ -1,8 +1,4 @@
 ALTER TABLE devices DROP CONSTRAINT devices_app_id_fkey;
-ALTER TABLE devices DROP CONSTRAINT devices_version_fkey;
-ALTER TABLE stats DROP CONSTRAINT logs_app_id_fkey;
-ALTER TABLE stats DROP CONSTRAINT logs_device_id_fkey;
-ALTER TABLE stats DROP CONSTRAINT logs_version_fkey;
```

Comment:
does this change take effect? considering this already ran

Reply:
Martin broke [base.sql](https://github.com/Cap-go/capgo/actions/runs/6512192143/job/17689348153) and the `supabase up` was no longer works (because of the broken base.sql). I fixed the base.sql and then it threw an exception in this migration so I removed the lines that were causing the problems (The problem was that `logs_app_id_fkey` constraint did not exist)